app/uk/gov/hmrc/helptosavefrontend/controllers/RegisterController.scala

```diff
@@ -99,7 +99,12 @@ class RegisterController @Inject() (val helpToSaveService:     HelpToSaveService
           logger.warn(s"Error while trying to create account: ${submissionFailureToString(e)}", nino)
           SeeOther(routes.RegisterController.getCreateAccountErrorPage().url)
         },
-          _ ⇒ SeeOther(frontendAppConfig.nsiManageAccountUrl)
+          submissionSuccess ⇒ submissionSuccess.accountNumber.fold(
+            SeeOther(frontendAppConfig.nsiManageAccountUrl)
+          ) {
+              acNumber ⇒
+                Ok(views.html.register.account_created(acNumber.accountNumber))
```

Comment:
the ticket talks about new response url `/help-to-save/account-created`

Reply:
you're right, I've missed that

Reply:
I'm sure Andy said on Friday that we don't need a URL for that page, I will check with Marcus and Nia